context/app/static/js/components/detailPage/related-entities/RelatedEntitiesSectionHeader/RelatedEntitiesSectionHeader.jsx

```diff
@@ -2,16 +2,23 @@ import React from 'react';
 import Button from '@material-ui/core/Button';
 
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+import { useFlaskDataContext } from 'js/components/Contexts';
 import { SpacedSectionButtonRow } from 'js/shared-styles/sections/SectionButtonRow';
 
-function RelatedEntitiesSectionHeader({ header, searchPageHref, iconTooltipText }) {
+const tooltipTexts = {
+  Donor: 'Samples and datasets derived from this donor.',
+  Sample: 'Datasets derived from this sample.',
+  Dataset: 'These datasets include those that have additional processing, such as visualizations.',
+};
+
+function RelatedEntitiesSectionHeader({ header, searchPageHref }) {
+  const {
+    entity: { entity_type },
+  } = useFlaskDataContext();
+
   return (
     <SpacedSectionButtonRow
-      leftText={
-        <div>
-          <SectionHeader iconTooltipText={iconTooltipText}>{header}</SectionHeader>
-        </div>
-      }
+      leftText={<SectionHeader iconTooltipText={tooltipTexts[entity_type]}>{header}</SectionHeader>}
```

Comment:
You could use optional chaining here as well? If you do, I'd double check.
```suggestion
      leftText={<SectionHeader iconTooltipText={tooltipTexts?.[entity_type]}>{header}</SectionHeader>}
```